graphqlapi/src/main/resources/schemafirst/cql_directives.graphql

```diff
@@ -114,6 +114,16 @@ directive @cql_index (
 
 ) on FIELD_DEFINITION
 
+"""
+The consistency level of the CQL SELECT generated for a query.
+"""
+enum QueryConsistency {
+    LOCAL_ONE
+    LOCAL_QUORUM
+    ALL
```

Comment:
We have `ALL`, but no `QUORUM`?

Reply:
I used the same levels as the CQL-first API, I wasn't there since the beginning so I don't know how they were decided.